### PR TITLE
Added back dataset filesize.

### PR DESCRIPTION
**Pull request by mickel1138**

### CODE REVIEW

**Review by praneethpr — APPROVED** [review]
Tested and works, stats also works

File: src/Entity/Fileset.php [hunk 1]
```diff
@@ -157,7 +157,7 @@ public function getFileSize() :int
     {
         $fileSize = 0;
 
-        foreach ($this->files as $file) {
+        foreach ($this->getProcessedFiles() as $file) {
```
Comment (fightingtexasaggie) [inline src/Entity/Fileset.php:160]: should we fix this in a different ticket? I guess since this is an epic, no biggie either way...

Comment (mickel1138) [reply src/Entity/Fileset.php:160]: It's something we forgot to fix before. I think what we want from this is the file size of all the actual files (processed) in a dataset, no the deleted or pending ones.

**Review by fightingtexasaggie — APPROVED** [review]
tested, works (after migration)

**Pull request merged**